lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -43,7 +44,7 @@ sub new {
     bless( $self, $type );
 
     if ( ! $params || ! $params->{config} ) {
-        handle_exception('new', "Missing 'config' parameter", '001');
+        handle_exception('new', "Missing 'config' parameter");
```

Comment:
Since the method name is not used anymore, is it still required to pass it to `handle_exception()`?